### ISSUE TITLE
Support for postgres data type json?

### ISSUE THREAD

**Issue by MaVoit** [simov/express-admin#147]
Is there any way I can display the value of a column with datatype json in the edit view?
Currently I am just getting the string representation of the object, which is [object Object].

settings.json:
```
            {
                "name": "custom_data",
                "verbose": "Data",
                "control": {
                    "text": true
                },
                "type": "json",
                "allowNull": true,
                "defaultValue": null,
                "listview": {
                    "show": true
                },
                "editview": {
                    "show": true
                }
            }
```

**Comment by yusufyildirim** [comment]
Hey, I was facing with the same issue and I tweaked the library a bit  :D In the end, I did this: https://github.com/yusufyildirim/express-admin/commit/daf359f09bc2cd7dcc5e06bac8ec70c59271f29b

This allows you to use "text" control but converts JSON objects to the string so you can see it in edit view. Propably, we need to add extra json view or something like that to edit our json easily but in my case, I don't need it for know because my json datas are simple arrays.

If you want help, please comment so maybe I can help you to improve this.

**Comment by MaVoit** [comment]
Yes, excellent tip. I was thinking of doing exactly the same, but was hoping that I wouldn’t have to fork the repo and that I was just missing something :)

**Comment by yusufyildirim** [comment]
I found a bug today. Sometimes, this function triggers for the same data and this behavior causes a problem. Assume you have a json array like `["A", "B", "C"]`, if you convert it to string with JSON.stringfy method, everything is allright but if you do the same thing again to that string, you found yourself with a data like `"[\"A\", \"B\", \"C\"]"`. To prevent this, I changed my code a bit. Here is my new approach: https://github.com/yusufyildirim/express-admin/commit/b6980f4f5ec0fe4810bc812d6736d31bea518427

**Comment by MaVoit** [comment]
👍🏻
